tuf/tuf.go

```diff
@@ -625,7 +616,7 @@ func (tr *Repo) UpdateSnapshot(role string, s *data.Signed) error {
 }
 
 // UpdateTimestamp updates the snapshot meta in the timestamp based on the Signed object
```

Comment:
We should also update this comment to indicate that this method can be used to set previous timestamp data for this timestamp.